include/deal.II/grid/grid_tools.h

```diff
@@ -3128,23 +3142,17 @@ namespace GridTools
 
 namespace GridTools
 {
-  // declare specializations
-  template <>
-  double
-  cell_measure<1>(const std::vector<Point<1>> &,
-                  const unsigned int (&)[GeometryInfo<1>::vertices_per_cell]);
-
-  template <>
-  double
-  cell_measure<2>(const std::vector<Point<2>> &,
-                  const unsigned int (&)[GeometryInfo<2>::vertices_per_cell]);
-
-  template <>
+  template <int dim>
   double
-  cell_measure<3>(const std::vector<Point<3>> &,
-                  const unsigned int (&)[GeometryInfo<3>::vertices_per_cell]);
-
-
+  cell_measure(
+    const std::vector<Point<dim>> &all_vertices,
+    const unsigned int (&indices)[GeometryInfo<dim>::vertices_per_cell])
+  {
+    // We forward call to the ArrayView version:
+    const ArrayView<const unsigned int> view(
+      &indices[0], GeometryInfo<dim>::vertices_per_cell);
```

Comment:
Can't we replace by `&indices[0]` by `indices`?

Reply:
yep, did that.